metadata.rb

```diff
@@ -5,12 +5,14 @@
 maintainer_email 'me@bdang.it'
 license          'MIT'
 description      'InfluxDB, a timeseries database'
-version          '6.3.1'
+version          '6.3.2'
 
 supports 'centos'
 supports 'debian'
 supports 'redhat'
 supports 'ubuntu'
+supports 'amazon'
+
```

Comment:
Can you remove line15? Rubocop caught this extra line.

Reply:
Hey @bdangit , done! :)

Reply:
Thanks!